contracts/0.8.25/vaults/dashboard/Permissions.sol

```diff
@@ -228,11 +228,23 @@ abstract contract Permissions is AccessControlConfirmable {
      */
     function confirmingRoles() public pure virtual returns (bytes32[] memory);
 
+    /**
+     * @dev A custom modifier that checks if the caller has a role or the admin role for a given role.
+     * @param _role The role to check.
+     */
+    modifier onlyRoleMemberOrAdmin(bytes32 _role) {
+        if (hasRole(_role, msg.sender) || hasRole(getRoleAdmin(_role), msg.sender)) {
```

Comment:
the only question is order here: what would be more favorable due to the short-circuiting calculations:
- to assign granular roles and pay less gas
- to not assign granular roles and pay less gas if put other way around

Reply:
I'd prefer the favor the granular roles, just to incentivize more secure setup